code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -69,27 +71,62 @@ public Account getAdminAccount() {
     }
 
     @Override
-    public List<Account> searchAccounts(String name) {
-        List<Account> accounts = new ArrayList<>();
-        accounts.addAll(create()
-                .selectFrom(ACCOUNT)
-                .where(ACCOUNT.STATE.eq(CommonStatesConstants.ACTIVE)
-                        .and(ACCOUNT.NAME.contains(name))
-                        .and(ACCOUNT.KIND.eq(AccountConstants.USER_KIND)
-                                .or(ACCOUNT.KIND.eq(AccountConstants.ADMIN_KIND))))
-                .orderBy(ACCOUNT.ID.asc()).fetch());
-        return accounts;
+    public List<Account> searchUsers(String username) {
+        try {
+            List<Account> accounts = new ArrayList<>();
+            accounts.addAll(create()
```

Comment:
Nit: instead of defining/populating List data structure here, you can just return the result of select directly:

```
return create().select..
``